### PR TITLE
add fr-FR translation for README.md

### PR DESCRIPTION
**Pull request by yezz123**

### CODE REVIEW

File: README.fr_FR.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+## Contact
+
+| Équipe                        | Contact                       |
+|-------------------------------|-------------------------------|
+| Presse                        | outreach@rockylinux.org       |
+| Développement                 | development@rockylinux.org    |
+| Infrastructure                | infrastructure@rockylinux.org |
+| Sécurité                      | security@rockylinux.org       |
+| Web                           | web@rockylinux.org            |
```
Comment (Addvilz) [inline README.fr_FR.md:50]: "Web et Branding" et ajoutez un autre email sur la même ligne comme "web@rockylinux.org, brand@rockylinux.com"

**Review by Addvilz — CHANGES_REQUESTED** [review]

File: README.fr_FR.md (hunk 1 above)
Comment (yezz123) [reply README.fr_FR.md:50]: Adding Branding Done! ✌

File: README.fr_FR.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+| Branding                      | brand@rockylinux.com          |
```
Comment (Addvilz) [inline README.fr_FR.md:51]: Je voulais dire par là https://github.com/rocky-linux/rocky/pull/49/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L50

**Review by Addvilz — CHANGES_REQUESTED** [review]

File: README.fr_FR.md (hunk 2 above)
Comment (yezz123) [reply README.fr_FR.md]: done !

Comment (yezz123) [reply README.fr_FR.md]: @Addvilz Check now if everything is good ?

File: README.md [hunk 3]
```diff
@@ -47,7 +47,7 @@ Please view the [contributing](#contributing) section below.
 | Development                   | development@rockylinux.org    |
 | Infrastructure                | infrastructure@rockylinux.org |
 | Security                      | security@rockylinux.org       |
-| Web                           | web@rockylinux.org            |
+| Web and Branding              | web@rockylinux.org, brand@rockylinux.com  |
```
Comment (Addvilz) [inline README.md:50]: Please, revert this. There is already a pull request for this. I did not ask you to do this.

**Review by Addvilz — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (yezz123) [reply README.md:50]: Okay ! ✌

**Comment by Addvilz** [comment]
Why did you delete the whole README.md?

**Pull request closed without merging**